presidio-image-redactor/README.MD

```diff
@@ -1,18 +1,101 @@
 # Presidio Image Redactor
 
+***Please notice, this package is still in alpha and not production ready.***
+
 ## Description
 
-The Presidio Image Redactor is a Python based module for detecting and redacting PII text
-entities in images.
+The Presidio Image Redactor is a Python based module for detecting and redacting PII
+text entities in images.
 
-## Usage
+## Installation
 
 Pre-requisites:
 
--   Install [Tesseract OCR](https://github.com/tesseract-ocr/tesseract) by following the instructions on how to install it for your operating system. The Image-Redactor was tested with a minimal version of 4.0.0
--   Install [Spacy model](https://spacy.io/usage/models) to use with the presidio-analyzer.
+- Install [Tesseract OCR](https://github.com/tesseract-ocr/tesseract) by following the
+  instructions on how to install it for your operating system.
+
+  The Image-Redactor was tested with a minimal version of 4.0.0
+- Install [Spacy model](https://spacy.io/usage/models) to use with the
+  presidio-analyzer.
+
+  Once Installed, run the following command to download the default spacy model needed
```

Comment:
Once the image-redact or is installed with pip, the user will have spacy installed, so no other action besides the model download is necessary.

Reply:
Done.